src/storage/examples/src/objects/upload_with_kms_key.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START storage_upload_with_kms_key]
+use google_cloud_storage::client::Storage;
+
+pub async fn sample(
+    client: &Storage,
+    bucket: &str,
+    object: &str,
```

Comment:
nit: and I feel like we have been hardcoding the name, e.g.

https://github.com/googleapis/google-cloud-rust/blob/26f779c551da5f348fd556a243796a79ee8b7a67/src/storage/examples/src/objects/stream_file_upload.rs#L20

Reply:
Done.